script.module.pyxbmct/addon.xml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyxbmct"
+   name="PyXBMCt"
+   version="1.3.1+matrix"
```

Comment:
`1.3.1+matrix.1` if possible, just for consistency with other script modules already on matrix